StylableSwiftUI/Classes/Stylist.swift

```diff
@@ -32,6 +32,11 @@ public class Stylist: ObservableObject {
             self.scoredStyleMatchCache = [:]
         }
     }
+    @Published public var currentTheme: Theme? {
+        didSet {
+            self.scoredStyleMatchCache = [:]
```

Comment:
How do we expect users to use this? Is there a need (future pr) for storing the caches of themes to help if users want to swap to/from themes often? Can't think of a use case for this, just a thought!

Reply:
Yeah, I'll see what performance is like when we implement our theme and add any performance optimizations if there are bottlenecks.